src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -2097,6 +2097,25 @@ void primitive_inst::execute() {
 
     set_out_event(_impl->execute(_impl_params->dep_events, *this));
 
+    GPU_DEBUG_IF(get_config().get_validate_buffer_finding_inf()) {
+        get_network().get_stream().finish();
+        auto &layout = _impl_params->get_output_layout(0);
+        auto output_mem = output_memory_ptr(0);
+        if (output_mem && layout.data_type == data_types::f16) {
+            mem_lock<ov::float16, mem_lock_type::read> lock(output_mem, get_network().get_stream());
+            for (size_t k = 0; k < lock.size(); k++) {
+                if (std::isinf(lock[k]) || std::isnan(lock[k])) {
+                    std::string iter;
+                    #ifdef GPU_DEBUG_CONFIG
```

Comment:
We don't need `#ifdef` here because we are already in `GPU_DEBUG_IF`

Reply:
Applied

Reply:
oh I see.. then, it would be easier to read if you could add like below.
```
#ifdef GPU_DEBUG_CONFIG
int64_t get_current_iteration_num() { return iteration; }
#else
int64_t get_current_iteration_num() { const int64_t NOT_AVAILABLE=112233; return NOT_AVAILABLE; }
#endif
```
Then we don't need `#ifdef` here.